### PR TITLE
Address comments regarding alpha for gateway topology features

### PR DESCRIPTION
**Pull request by jacob-delgado**
Update from comments in https://github.com/istio/enhancements/issues/20

* Added automated tests
* Updated istio.io docs

### CODE REVIEW

File: features/configuring_gateway_network_topology.md
Comment (brian-avery) [inline features/configuring_gateway_network_topology.md:124]: @jacob-delgado is this ready for alpha? It looks like everything is checked at this point. If so, can you also update [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml)?

Comment (jacob-delgado) [reply features/configuring_gateway_network_topology.md:124]: Updated features.yaml

**Comment by jacob-delgado** [comment]
Waiting on final TOC approval, but I've done all of the work necessary to move this into Alpha.

**Comment by jacob-delgado** [comment]
Approved on TOC at 4/19

See https://docs.google.com/document/d/13lxJqtlaQhmV2EwsNnS6h-_O4pobZQZuMjrzOeMgVI0/edit

**Review by brian-avery — APPROVED** [review]

**Comment by ericvn** [comment]
@jacob-delgado I believe that this needs to be in the 1.10 branch as you are expecting it for the 1.10 doc changes.

**Pull request merged**
